### ISSUE TITLE
V3.9 Lampe schalten kein push für level der Lampe

### ISSUE THREAD

**Issue by Baenker** [iobroker-community-adapters/ioBroker.hue#414]
Ich mache mal ein neues Issue auf. 

Also Hue Adapter auf debug. Eine Gruppe mit einer Lampe wird eingeschaltet per App. Level wird erst nach den poll angepasst. Wenn ich das Log richtig interpretiere kommt beim push Brightness rein und eben der Datenpunkt on.

Im Log ist einmal ein und einmal ausschalten

[debug-log_hue.txt](https://github.com/iobroker-community-adapters/ioBroker.hue/files/11719391/debug-log_hue.txt)

**Comment by foxriver76** [comment]
In dem Log finde ich nur updates die `brightness` enthalten für `grouped_light` d.h. für Gruppen. 

Das normale `light` im Log sendet nur `on`. Ist das eine originale Hue Lampe?

**Comment by Baenker** [comment]
ja, eine normale mit E27 Fassung. Wie gesagt geschaltet habe ich über die App die Gruppe  (=Raum) in der nur die einzelne Lampe ist. Dort einfach nur "Ein". Also keine Szene.

Was ist eigentlich der Unterschied zwischen Brightness und Level? Außer das Brighness bis 255 geht?

**Comment by foxriver76** [comment]
Gibt von der Logik keinen Unterschied, Level ist in %, Brightness ein 8 Bit Wert. 

Hm.. wenn du die Lampe direkt schaltest, ohne Gruppe bekommst du dann ein direktes Update? So habe ich es bei mir getestet gehabt.

**Comment by Baenker** [comment]
Nein auch dann nicht. Das resultat ist gleich.

**Comment by mcm1957** [comment]
Issue seems to be outdated.
If the problem still exists with current release (3.17.2) please open a new release.
